app/clients/freshdesk.py

```diff
@@ -96,8 +104,34 @@ def send_ticket(self) -> int:
 
                 return response.status_code
             else:
+                self.email_freshdesk_ticket(self._generate_ticket())
                 return 201
-        except requests.RequestException as e:
+        except requests.RequestException:
             content = json.loads(response.content)
-            current_app.logger.error(f"Failed to create Freshdesk ticket: {content['errors']}")
-            raise e
+            current_app.logger.error(f"Failed to create Freshdesk ticket: {content}")
+            self.email_freshdesk_ticket(self._generate_ticket())
+            return 201
+
+    def email_freshdesk_ticket(self, content: dict) -> None:
```

Comment:
Do you think we should catch further errors that might trigger from sending the notification and log?

Reply:
If there are further errors saving or sending the ticket we want that to surface as an error.

Hrm. we could log the contact info in this case. Might have PII in it, but we could delete the log stream later. Likely better than throwing away the contact data. Probably we'd want to raise a critical alarm as well, some sort of "Contact data could not be sent" so we'd know to check the logs.

Thoughts?

Reply:
Sounds good to me. I think we can drop the re-raise though, else we'll get this error twice in the logs (and we are not sure how/what the parent exception handler might show). The exception handled, so we can consider it properly managed and move on. We could then add alarms on these logs to surface it up to #notification-ops . Does that make sense?

Reply:
👍  yeah. And the main thing is that we're logging that the contact email failed.